index.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 
-var net = require('net');
-var fs = require('fs');
-var util = require('util');
-var path = require('path');
-var jsome = require('jsome');
-var options = require('./util/usage').options;
-var settings = require('./settings.json');
+const net = require('net'),
+    fs = require('fs'),
+    util = require('util'),
+    path = require('path'),
+    jsome = require('jsome'),
+    colors = require('colors'),
+    jsbytes = require('jsbytes'),
```

Comment:
Where is this used?

Reply:
This is going to be used instead of some things ByteBuffer does; as I mentioned in my PR comment, I am in the middle of removing ByteBuffer deps

- Xzlash

Reply:
Also I am aware that some styling is inconsistent, I will fix this once I fix ByteBuffer